core/vm/actor/builtin/types/miner/v2/monies.hpp

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+#include "vm/actor/builtin/types/miner/v0/monies.hpp"
+
+namespace fc::vm::actor::builtin::v2::miner {
+  using fc::common::math::kPrecision128;
+  using fc::common::smoothing::FilterEstimate;
+  using fc::primitives::ChainEpoch;
+  using fc::primitives::StoragePower;
+  using fc::primitives::TokenAmount;
+  using fc::vm::actor::builtin::states::MinerActorState;
+  using fc::vm::actor::builtin::types::miner::kRewardVestingSpecV1;
+  using fc::vm::actor::builtin::types::miner::VestSpec;
+  using fc::vm::runtime::Runtime;
+  using fc::vm::version::NetworkVersion;
+
+  class Monies : public v0::miner::Monies {
+   public:
+    struct BigFrac {
+      BigInt numerator;
+      BigInt denominator;
+    };
+
+    BigInt initial_pledge_max_per_byte =
+        BigInt(1e18) / BigInt(BigInt(32) << 30);
```

Comment:
```suggestion
        bigdiv(1000000000000000000, BigInt{32} << 30);
```

Reply:
@Elestrias, you should to use `bigdiv()` in all places when you see `Big.Div()` in go